job_definitions/code_backups.yml

```diff
@@ -57,6 +58,7 @@
       - ansicolor
     builders:
       - shell: |
+          cd ${WORKSPACE}
```

Comment:
Why is this necessary?

Reply:
Sorry, I pushed by accident whilst I was testing different approaches. I will force push once I have a better solution.

Reply:
OK - let me know when you want me to take a proper look.